tests/test_blend.py

```diff
@@ -270,12 +270,12 @@ def test_get_blend_forecast_values_two_models_plevel_second(db_session):
     assert forecast_values_read[4].expected_power_generation_megawatts == 1.5  # mix
     assert forecast_values_read[5].expected_power_generation_megawatts == 1  # test_1
 
-    assert forecast_values_read[0]._properties == {"10": 0.9, "90": 1.1}
-    assert forecast_values_read[1]._properties == {"10": 0.9, "90": 1.1}
-    assert forecast_values_read[2]._properties == {"10": 1.8, "90": 2.2}
-    assert forecast_values_read[3]._properties == {"10": 1.8, "90": 2.2}
-    assert forecast_values_read[4]._properties == {"10": 1.35, "90": 1.65}
-    assert forecast_values_read[5]._properties == {"10": 0.9, "90": 1.1}
+    assert forecast_values_read[0]._properties == {"10": 0.9, "90": 1.1} or forecast_values_read[0]._properties == {}
```

Comment:
I think it would be good to write a similar test, and check `_properties == {}`

Reply:
You mean for the remaining lines? 
276 277?

Reply:
yea. feels like we should set up a seperate test where there are no _properties, and check we get a {} back. Then we can really, double sure check it works, and in the future make sure it doesnt break

Reply:
> yea. feels like we should set up a seperate test where there are no _properties, and check we get a {} back. Then we can really, double sure check it works, and in the future make sure it doesnt break

hi @peterdudfield ive added an additional test, is that ok?